source/_integrations/workday.markdown

```diff
@@ -66,31 +65,48 @@ add_holidays:
 {% endconfiguration %}
 
 Days are specified as follows: `mon`, `tue`, `wed`, `thu`, `fri`, `sat`, `sun`.
-The keyword `holiday` is used for public holidays identified by the holidays module.
+The keyword `holiday` is used for public holidays identified by the holiday’s module.
 
 <div class='note warning'>
 
 If you use the sensor for Norway (`NO`) you need to wrap `NO` in quotes or write the name in full.
-Otherwise the value is evaluated as `false`.
+Otherwise, the value is evaluated as `false`.
 If you use the sensor for Canada (`CA`) with Ontario (`ON`) as `province:` then you need to wrap `ON` in quotes.
-Otherwise the value is evaluated as `true` (check the YAML documentation for further details) and the sensor will not work.
+Otherwise, the value is evaluated as `true` (check the YAML documentation for further details) and the sensor will not work.
+
+> **_NOTE:_**
```

Comment:
This is not how we do notes, besides it is a huge blob of text, which makes it hard to read.

Please check our developer documentation on how to make alerts/notes, and see if you can improve on this paragraph a bit to make it more readable.